packages/lib/src/vis/scatter/ScatterVis.tsx

```diff
@@ -104,9 +104,7 @@ function ScatterVis(props: Props) {
         }}
         title={title}
       >
-        <DefaultInteractions
-          interactions={{ XAxisZoom: false, YAxisZoom: false, ...interactions }}
-        />
+        <DefaultInteractions withAxialZoom={false} {...interactions} />
```

Comment:
Note that previously, we were turning off wheel-based axial zoom completely but not selection-based axial zoom. I wasn't sure if this was intended, so now all axial zoom interactions are off. I have no strong feeling about this and `withAxialZoom`, so let me know what you think.

Reply:
Not sure why I decided to remove axial zoom. I reckon that it can be enabled for `ScatterVis`.

Reply:
Yeah alright, I'll remove the `withAxialZoom` prop as well, what do you reckon?

Reply:
Ah just saw your other comment :+1: